client/extensions/woocommerce/app/settings/payments/payment-method-item-placeholder.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ListItem from 'woocommerce/components/list/list-item';
+import ListItemField from 'woocommerce/components/list/list-item-field';
+
+class PaymentMethodItemPlaceHolder extends Component {
```

Comment:
Since this is a basic component with just markup, can you update it to be a stateless component?